cypress/component/DefendantDetails.cy.tsx

```diff
@@ -102,7 +102,7 @@ describe("Defendant Details", () => {
 
     cy.mount(
       <CourtCaseContext.Provider value={{ courtCase }}>
-        <DefendantDetails amendFn={() => () => {}} amendmentRecords={{}} />
+        <DefendantDetails amendFn={() => () => {}} amendmentRecords={{}} stopLeavingFn={() => {}} />
```

Comment:
Am I wrong in thinking that we now have two standards for this? `onAmend` `onLeave` vs. above